components/gc-minister/index.json-ld

```diff
@@ -164,6 +168,67 @@
 					}
 				]
 			}
+		},
+		{
+			"@id": "_:implement_minister_2",
+			"iteration": "_:iteration_minister_2",
+			"name": {
+				"en": "Standard",
+				"fr": "Standard"
+			},
+			"introduction": {
+				"en": "This implementation is meant for developers/publishers adding the component manually.",
+				"fr": "Cette implémentation est destinée aux développeurs/éditeurs qui ajoutent le composant manuellement."
+			},
+			"instructions": {
+				"en": [
```

Comment:
Instructions should reflect only what to do to migrate from the previous version to this new version.